src/events/order/cd-event-add-child-ticket.vue

```diff
@@ -21,6 +21,7 @@
                         :proportions="[2, 2, 3]" data-vv-value-path="value" :data-vv-name="`dob-${id}`" v-validate="'required'"></vue-dob-picker>
       </div>
       <p class="cd-child-ticket__dob-err text-danger" v-show="errors.has(`dob-${id}:required`)">{{ $t('Date of birth is required') }}</p>
+      <p class="cd-child-ticket__dob-u19-err text-danger" v-show="errors.has(`dob-u19-${id}:overage`)">{{ $t('Youths need to be 18 years old or younger.') }}</p> 
```

Comment:
Update error message
Also needs translation

Reply:
I think the error msg should be 'Youth is overage (19+)'. Yep I do translations last, that will be done :)

Reply:
Ooohhh... *Over age*. _Overage_ is
> an excess or surplus, especially the amount by which a sum of money is greater than a previous estimate.